src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -32,7 +32,6 @@ case class OneWorkerSpecifiedInClusterRequestException()
 // this needs to be a Universal Trait to enable mixin with Value Classes
 // it only serves as a marker for StringValueClassFormat
 sealed trait StringValueClass extends Any
-case class GoogleServiceAccount(string: String) extends AnyVal with StringValueClass
```

Comment:
Should `GoogleServiceAccount` in `automation` be removed as well?

https://github.com/broadinstitute/leonardo/blob/aa152d4c2fb415144787de141fbd82301c4eb4e5/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala#L14

Reply:
Sure - but my preferred solution is for Leo-Automation to depend on a published Leo Model, so I am not sure there is value in cleaning up LeonardoModelCopy before then

Reply:
ok, fair enough.